cpp/src/Ice/HashUtil.h

```diff
@@ -7,50 +7,60 @@
 
 #include <cstddef>
 #include <cstdint>
+#include <functional>
 #include <map>
 #include <string>
 #include <vector>
 
+#include "EndpointI.h"
```

Comment:
We usually put our includes at the top, and std after ours.

Reply:
Fixed